src/gui/addnewtorrentdialog.ui

```diff
@@ -271,6 +271,14 @@
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
+             <property name="geometry">
```

Comment:
Is it intended to be?

Reply:
qt designer added those, I suppose we should keep it?

Reply:
>qt designer added those, I suppose we should keep it?

I have repeatedly come across the fact that Qt Designer/Creator changes the geometry settings for some widgets, but it seems that their values depend on the Qt (Designer/Creator) version, platform, or something else. So I prefer to delete such unintended changes. Otherwise, they may be commited with each change of `.ui` files.

Reply:
removed.